requirements.txt

```diff
@@ -1,2 +1,4 @@
-tf-nightly-2.0-preview
+tf-nightly
```

Comment:
why change this back?

Reply:
TFP-nightly and tf-nightly don't seem to work together. tfp-nightly trys to import tf.contrib which from my understanding, and what the importerror tells me, not available anymore.